package.json

```diff
@@ -51,13 +51,19 @@
     "*.{js,jsx,ts,tsx}": "eslint"
   },
   "dependencies": {
+    "base-x": "^1.0.4",
+    "bs58": "^4.0.1",
```

Comment:
`bs58` and it's types aren't needed anymore with the new approach.

Reply:
Sorry, I thought I removed those. Thanks for catching